view/frontend/web/click_conversion_analytics.js

```diff
@@ -42,4 +63,57 @@ requirejs(['algoliaBundle', 'algoliaAnalytics'], function(algoliaBundle, algolia
 		
 		return search;
 	});
-});
\ No newline at end of file
+	
+	function trackClick(objectID, position) {
+		objectID = objectID.toString();
+		
+		var clickedObjectIds = getObjectIds('clicked');
+		if (!clickedObjectIds[objectID]) {
+			algoliaAnalytics.click({
+				objectID: objectID.toString(),
+				position: parseInt(position)
+			});
+			
+			clickedObjectIds[objectID] = 1;
```

Comment:
This kind of array management with numeric indices creates a lot of null values, use a non numeric ("product-[ID]") instead ?